### PR TITLE
Extractor version doc

### PR DESCRIPTION
**Pull request by kukovecz**
Documenting extractors

### CODE REVIEW

**Review by QKaiser — CHANGES_REQUESTED** [review]
I'd suggest a table listing all the extractors currently in use with the version we recommend to use for each of them. This way we can point to it if we receive complaints from users who use old versions of 7z :)

This table should also point out whether it is automatically installed as part of Python dependency management (e.g. ubireader, jefferson) or needs to be installed manually.

File: README.md [hunk 1]
```diff
@@ -24,6 +24,17 @@ Help on usage:
 docker run --rm --pull always ghcr.io/iot-inspector/unblob:latest --help
 ```
 
+## Extractors
+
+Unblob relies on various tools for extracting the contents of a blob. These extractors are either developed or externally using 3rd party tools, or internally (`unblob/extractors` directory). \
```
[the remark this answers is not in the document]
Comment (kukovecz) [reply README.md:29]: God, I was phrasing this for too much time and still managed to write this abomination. Thank you for fixing it up!

File: README.md [hunk 2]
```diff
@@ -24,6 +24,17 @@ Help on usage:
...
+To be able to use unblob properly, all extractors needs to be installed.
+
+Hints for extractor installation:
+* If you are using unblob from the official docker container, nothing to be done.
+* Check the `Dockerfile` how different extractors are installed (keeping mind of the extractor versions).
```
[the remark this answers is not in the document]
Comment (kukovecz) [reply README.md]: Great idea!

**Comment by kukovecz** [comment]
> I'd suggest a table listing all the extractors currently in use with the version we recommend to use for each of them. This way we can point to it if we receive complaints from users who use old versions of 7z :)
> 
> This table should also point out whether it is automatically installed as part of Python dependency management (e.g. ubireader, jefferson) or needs to be installed manually.


I was lazy enough not to put a table like this consciously :sweat_smile: 
I just don't want the burden for the development by continuously updating the README, or make it possible to be outdated. But yeah, putting those versions explicitly here would maybe help the user more. 

Let's have a small brainstorm with the others what to do with this.

**Comment by vlaci** [comment]
I'd put in the versions from the Ubuntu image we are currently using for tests (plus the 7z we are downloading). If we are to update the image, we need to update the docs as well but I don't think that it will be that frequent of an occasion.

**Review by QKaiser — APPROVED** [review]

**Pull request merged**